js/narrativeView.js

```diff
@@ -185,7 +185,21 @@ define([
 
       if (this._isInitial) return;
 
-      const $elementToFocus = this.isLargeMode() ? this.$(`.narrative__content-item[data-index="${itemIndex}"]`) : this.$('.narrative__strapline-btn');
+      const hasStraplineTransition = !this.isLargeMode() && parseFloat($straplineHeaderElm.css('transitionDuration'));
```

Comment:
```suggestion
      const hasStraplineTransition = !this.isLargeMode() && ($straplineHeaderElm.css('transitionDuration') !== '0s');
```
Sorry to be a complete * cough * but @tomgreenfield has convinced me that your original comparison was more appropriate.
It results in a boolean and `$straplineHeaderElm.css('transitionDuration')` is always normalised to `0s` for `0ms` or `0.0000ms` or `undefined`.